docs/datatype/image/text2image.md

```diff
@@ -1,3 +1,479 @@
 # Search Image from Text via CLIP model
 
-TBA
\ No newline at end of file
+In this tutorial, we will create an image search system that retrieves images based on short text descriptions as query.
+
+The interest behind this is that in regular search, image description or meta data describing the content of the image needs to be indexed first before retrieving the images via text query. This can be expensive because you need a person to write that description and also information about image content is not always available.
+
+We need to look for another solution! What if we can directly compare text with images?  
+
+To do so, we need to figure out a way to match images and text. One way is finding related images with similar semantics to the query text. This requires us to represent both images and query text in the same embedding space to be able to do the matching. In this case, pre-trained cross-modal models can help us out.
+
+For example when we write the word "dog" in query we want to be able to retrieve pictures with a dog solely by using the embeddings similarity.
+
+```{tip}
+The full source code of this tutorial is available in this [google colab notebook](https://colab.research.google.com/drive/1_dsJA3rxdo4g68jmqJBUeUIVH6kAXQ05?usp=sharing)
```

Comment:
- Google Colab
- Can we host the notebook on GitHub, then import into Colab and share that? (That's what I did for https://colab.research.google.com/github/jina-ai/workshops/blob/main/memes/meme_search.ipynb) - makes it easier for others to fork